.github/workflows/codecov.yml

```diff
@@ -27,6 +27,7 @@ jobs:
       run: |
         sudo apt-get install software-properties-common
         sudo apt-add-repository ppa:opm/ppa
```

Comment:
Should this still be included?

Reply:
Yes, I removed it in my first commit. But `resinsight` is is install from that repo `ppa:opm/ppa`